api_tests/tests/ether_halight.ts

```diff
@@ -3,36 +3,70 @@
  * @ledger lightning
  */
 
-import { twoActorTest } from "../src/actor_test";
 import SwapFactory from "../src/actors/swap_factory";
 import { sleep } from "../src/utils";
+import { twoActorTest } from "../src/actor_test";
+
+describe("E2E: Ethereum/ether - Lightning/bitcoin", () => {
```

Comment:
Why did you introduce `describe` blocks again? 

I feel like they just add a level of indentation to the file and don't provide any value outside of that.

Reply:
Oh, I thought that was still our best practice. Will remove